### DISCUSSION TITLE
Python API documentation for different carla.Landmark.types and name

### DISCUSSION THREAD

**Discussion by EikReichma** [carla-simulator/carla#7218] [Q&A]
Hi, im quite new to carla and I need some help!
Im using the PythonAPI and my goal is to collect a large number of image data containing carla traffic signs. My initial idea was to find the location of all the traffic signs on a given map and spawn sensors relative to those location.
I know I can get landmarks by their type but I cannot find any documentation of which types exist and what their semantic meaning is. From code exploration I found that speed signs have the type='274' with subtype corresponding to the speed.
Where can I find this information and is my approach even a good idea?

I appreciate every comment :)

**Comment by Daraan** [comment]
Does this help?
https://carla.readthedocs.io/en/latest/python_api/#carla.LandmarkType

**Reply by EikReichma** [reply]
Yes it does help and it's a good start, thank you! However, I still don't know where to find information about the subtypes and other types, the LandmarkType only lists the most common ones. I tried to find it in the OpenDrive standard but I didn't have any luck. Maybe I'm looking in the wrong place.
